.pre-commit-config.yaml

```diff
@@ -16,3 +16,9 @@ repos:
     rev: v0.9.0.6
     hooks:
     -   id: shellcheck
+- repo: https://github.com/compilerla/conventional-pre-commit
+  rev: v3.1.0
+  hooks:
+    - id: conventional-pre-commit
+      stages: [commit-msg]
+      args: [--force-scope, misc, build, ci, docs, feat, fix, perf, refactor, style, test]
```

Comment:
Nack; I'm fine with some basic checks (like is the title short enough, are lines wrapped properly, etc.), but I don't want to force title modifiers on the project.

Reply:
You marked this as resolved, but didn't actually seem to change it?

Reply:
oof, apologies for that, thanks for catching this, now I've gone ahead and removed the `--force-scope` flag. Thanks.

Reply:
Genuinely resolved now, thanks.